### DISCUSSION TITLE
AudioGridder 1.2 issue

### DISCUSSION THREAD

**Discussion by doctormidi** [apohl79/audiogridder#506] [Help]
I use the plugin on Mac intel Mojave and the server on windows 10 system.AudioGridder 1.1 work perfect.And waiting for this update.specially multi out .And for me I want to thank you for this dream software . Biggg thnxxx .But when I install the 1.2 beta nothing work.
All plug-in format without any signal midi and audio.
Thnx for your help in advance.

**Comment by CaBuffalo** [comment]
Hi, doctormidi:

Please follow the instructions to report the problem you have. Simply
"nothing work" won't help us. Thanks.

https://audiogridder.com/bug-reports/

On Tue, Apr 20, 2021 at 1:30 PM doctormidi ***@***.***> wrote:

> I use the plugin on Mac intel Mojave and the server on windows 10
> system.AudioGridder 1.1 work perfect.And waiting for this update.specially
> multi out .And for me I want to thank you for this dream software . Biggg
> thnxxx .But when I install the 1.2 beta nothing work.
> All plug-in format without any signal midi and audio.
> Thnx for your help in advance.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/506>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6DTMSLD3U67LPL76ODTJXP6TANCNFSM43I77P6A>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by apohl79** [comment]
1.2 does currently only transmit data when playing/recording. This will change back to 1.1 behaviour in the next beta. Sounds like this is your issue.

If not, please submit a bug report as @CaBuffalo explained.

**Reply by doctormidi** [reply]
Okk..I use studio one and Logic Pro x.
I try all the format on a studio One.
VST3 : I try Kontakt it open on the server windows System and on the plugin on the Mac,but no audio Chanel is opened as usual when we open any vst instruments ,so no audio signal and the plugin don’t receive midi info.and no multi out option.
AU : also I try kontakt it open an audio Chanel but also no midi info so no audio signal and no multi out option.
VST : also the same but it open an audio Chanel and there is multi out option,but no midi signal and so no also an audio signal.
Thnx for your help..
